### PR TITLE
data(opening): new SEO job opening

### PR DESCRIPTION
**Pull request by Lalisfeed**
[Job update] : Added Senior SEO Specialist Job

I've added new job role to src/data/openings.json in reference to #1946

### CODE REVIEW

**Comment by Lalisfeed** [comment]
This is a pr in reference to #1946

File: src/data/openings.json [hunk 1]
```diff
@@ -36,6 +36,32 @@
     ],
     "applicationUrl": "https://docs.google.com/forms/d/e/1FAIpQLSf4Mk-vxO3Nl193LkCSGPOrrT5uDvA27peH8du7hSnfUsxb0A/viewform"
   },
+  {
+    "openingId": "senior-seo-specialist",
+    "category": "Engineering",
```
Comment (manishram) [inline src/data/openings.json:41]: Please change the category to **Marketing** for this role

**Review by manishram — CHANGES_REQUESTED** [review]

File: src/data/openings.json (hunk 1 above)
Comment (Lalisfeed) [reply src/data/openings.json]: Yes, I have updated it.

**Review by jamessspanggg — CHANGES_REQUESTED** [review]
`package-lock.json` does not need to be modified

**Comment by Lalisfeed** [comment]
> `package-lock.json` does not need to be modified

I think its fine for now. I have updated my commit

**Review by jamessspanggg — APPROVED** [review]

**Pull request merged**